src/components/thumbnail/document-type-collection.sass

```diff
@@ -144,11 +147,26 @@ $section-padding: 6px
         justify-content: center
 
         .icon-delete-document
-          height: 16px
-          width: 16px
-          fill: #808080
+          height: 23px
+          width: 23px
+          position: relative
+          top: 6px
+          right: -1px
+          border-radius: 4px
+          g
+            transform: translate(2px, 1.5px)
```

Comment:
It seems unusual and fragile to push transforms into particular elements inside an SVG.  Any reason that this couldn't be done just by modifying the transform that is on that `g` element in the .svg file, or by tweaking the 'top' and 'right' positioning?

Reply:
Thanks, that's a good point.  I didn't see the transform in the svg.  I tweaked it in there instead.  The reason for my transform was to center the g within the svg itself, which has a background that needs to change color on hover.  I used the new icon and colors specified in the zeplin.